src/main/java/com/hubspot/maven/plugins/dependency/management/DependencyManagementAnalyzer.java

```diff
@@ -26,7 +33,23 @@ public boolean analyze() {
     boolean success = checkDependencyManagement();
     // don't combine with previous line, we don't want short-circuit evaluation
     success &= checkPluginManagement();
-
```

Comment:
Nit: re-add this blank line. It's nice to separate things into logical code blocks.